Makefile

```diff
@@ -46,3 +45,38 @@ python-bootstrap: venv
 
 venv:
 	test -d venv || virtualenv venv
+
+memorydiskserver-docker-image:
```

Comment:
I would prefer this target was a script in the scripts folder and the makefile calls the script.